### PR TITLE
[docs] 2.1 DB1

### PR DESCRIPTION
**Pull request by jamiltz**
This change is [Reviewable](https://reviewable.io/reviews/couchbase/couchbase-lite-net/1021)

### CODE REVIEW

File: docs/modules/ROOT/pages/csharp.adoc [hunk 1]
```diff
@@ -15,17 +15,9 @@ Create or open an existing Virtual Studio project and install Couchbase Lite usi
...
-Install the `Couchbase.Lite` package.
-+
-*Couchbase Lite Enterprise Edition*
-+
-If you have a license or you are evaluating the Enterprise edition, install the `Couchbase.Lite.Enterprise` package.
-+
-. Your app must call the relevant `Activate()` function inside of the
+* Add `http://mobile.nuget.couchbase.com/nuget/Developer/` to your Nuget package sources (for prerelease only).
+
+Your app must call the relevant `Activate()` function inside of the
```
Comment (jamiltz) [inline docs/modules/ROOT/pages/csharp.adoc]: It's going to easy to forget to update this paragraph when switching between DB to GA in the cycle. Will try to add both.

**Comment by jamiltz** [comment]
@borrrden here some stubs for the 2.1 docs, could you add the c# examples?

**Comment by borrrden** [comment]
Updated!

**Comment by jamiltz** [comment]
@borrrden ready to merge.

**Pull request merged**